src/mongocxx/bulk_write.hpp

```diff
@@ -89,6 +89,13 @@ class MONGOCXX_API bulk_write {
     void append(const model::write& operation);
 
     ///
+    /// Gets a handle to the underlying implementation.
+    ///
+    /// @return Pointer to implementation for this object.
```

Comment:
Also, add a note mentioning that the returned pointer is only valid for as long as this object is?